packages/mongo/collection.js

```diff
@@ -90,7 +97,23 @@ Mongo.Collection = function Collection(name, options) {
   else if (Meteor.isClient) this._connection = Meteor.connection;
   else this._connection = Meteor.server;
 
-  if (!options._driver) {
+  this.openDriver = (driver) => {
+    this._driver = driver;
+    this._collection = this._driver.open(name, this._connection);
+  }
+
+  this.isAsync = options.isAsync;
+
+  if (options._driver) {
+    if (typeof options._driver.open !== 'function') {
+      throw new Error('If you are creating the driver manually using new ' +
+        'MongoInternals.RemoteCollectionDriver then you need to use ' +
+        'Promise.await() or await on it. Since it is async in recent ' +
+        'versions of Meteor. ' +
+        'Read more https://docs.meteor.com/changelog.html.');
```

Comment:
This should point to a specific docs article. Even better an upgrade guide in Guide.